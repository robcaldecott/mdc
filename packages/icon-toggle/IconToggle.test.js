import React from "react";
import { IconToggle } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(renderer.create(<IconToggle>favorite</IconToggle>)).toMatchSnapshot();
});

it("renders disabled", () => {
  expect(
    renderer.create(<IconToggle disabled>favorite</IconToggle>)
  ).toMatchSnapshot();
});
