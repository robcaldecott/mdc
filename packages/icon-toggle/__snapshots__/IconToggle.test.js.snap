// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<i
  className="mdc-icon-toggle material-icons"
>
  favorite
</i>
`;

exports[`renders disabled 1`] = `
<i
  className="mdc-icon-toggle material-icons mdc-icon-toggle--disabled"
>
  favorite
</i>
`;
